templates/security/adherent_login.html.twig

```diff
@@ -27,7 +27,12 @@
 
                     <a href="{{ path('forgot_password') }}" class="form__forgotten-pw">Oublié ?</a>
                 </li>
-
+                <li>
+                    <div class="b__nudge--top text--small form__checkbox form__checkbox--large" id="field-conditions">
+                        <input type="checkbox" id="remember_me" name="_remember_me"/>
+                        <label class="form form__label" for="remember_me">Se souvenir de  moi</label>
```

Comment:
There is a useless whitespace in the text

Reply:
fixed, thank yoy @thomasbisignani :D